MapboxDirections/MBRouteLeg.swift

```diff
@@ -34,15 +38,18 @@ open class RouteLeg: NSObject, NSSecureCoding {
             if let speed = jsonAttributes["speed"] {
                 segmentSpeeds = speed as? [CLLocationSpeed] ?? []
             }
-            if let nodes = jsonAttributes["nodes"] {
-                openStreetMapNodeIdentifiers = nodes as? [Int64] ?? []
+            if let congestion = jsonAttributes["congestion"] as? [String] {
+                congestionLevels = congestion.map {
+                    CongestionLevel(description: $0)!
```

Comment:
If we need to accept arbitrary congestion values, let’s handle that as tail work.